framework/src/media/utils/MediaUtils.cpp

```diff
@@ -322,14 +322,13 @@ bool header_parsing(FILE *fp, audio_type_t audioType, unsigned int *channel, uns
 				ret = fseek(fp, -1, SEEK_CUR);
 				if (ret != OK) {
 					meddbg("file seek failed errno : %d\n", errno);
-					free(header);
```

Comment:
same issue for line 408, the code may arrive there with a uninitialized header pointer.

Reply:
ah there is else case to avoid that. plz ignore.